doc/quickstart/geoext_quickstart.rst

```diff
@@ -12,29 +12,30 @@
 
 GeoExt is a JavaScript library that runs in the browser, it combines and enhances the ExtJS framework and OpenLayers library. 
 This quick start details the steps to create a rich web application using GeoExt components in less than 200 lines of code. 
-We will be using the installed MapServer web services on OSGeoLive a MapServer to supply map layers to the application. Completing the 
-steps below will result in the demo GeoExt application at http://localhost/geoext/. 
+We will be using the installed MapServer web services on OSGeoLive a MapServer to supply map layers to the application. Completing the steps below will result in the demo GeoExt application at http://localhost/geoext/. 
 
-HTML Page
----------
+.. contents:: Contents
+   :local:
 
-In this step we will set-up an empty HTML page which will contain the application. 
+Create an HTML page
+===================
 
-First create a new HTML file at ``/var/www/html/geoext/demo.html`` using the text editor Leafpad installed on OSGeoLive. To
-ensure you have permissions to save the file open Leafpad from the command line with elevated permissions using the
+In this step we will set-up an empty HTML page which will contain the application. We'll use the text editor Leafpad installed on OSGeoLive to create a new HTML file at ``/var/www/html/geoext/demo.html`` 
+
+#. Open Leafpad from the command line with elevated permissions (so you can save the file) using the
```

Comment:
@geographika Any idea why these "numbers" are displaying as hash symbols rather than numbers? Argh! I can see your tip about indenting the codeblocks has worked in the later sections, thanks!

Reply:
@flicstar - see https://github.com/flicstar/OSGeoLive-doc/pull/2 - the text also has to be aligned with the block.